### ISSUE TITLE
module not found: com.sksamuel.scapegoat#scalac-scapegoat-plugin_2.12;1.1.0

### ISSUE THREAD

**Issue by ikr0m** [sksamuel/sbt-scapegoat#53]

**Comment by vi-kas** [comment]
You may refer to [issue 45](https://github.com/sksamuel/sbt-scapegoat/issues/45) and see if this helps.

**Comment by ikr0m** [comment]
@vi-kas Thanks. #45 is very similar issue to mine. I've got subprojects and sbt `scapegoat` task gives `module not found: com.sksamuel.scapegoat#scalac-scapegoat-plugin_2.10;1.0.0` error. I'm actually using scala 2.12 and sbt 0.13.15 versions. I'm trying to fix it.

**Comment by vi-kas** [comment]
You may try to override your buildSettings for scapegoatVersion, something like:
override def buildSettings = super.buildSettings ++ Seq(scapegoatVersion := "1.3.2")
Then for projectSettings give scapegoatVersion := { (scapegoatVersion in ThisBuild).value }

by default it's taking scapegoatVersion as 1.0.0. See [ScapegoatSbtPlugin.scala](https://github.com/sksamuel/sbt-scapegoat/blob/master/src/main/scala/com/sksamuel/scapegoat/sbt/ScapegoatSbtPlugin.scala)

**Comment by ikr0m** [comment]
Can you point where can I add `override def buildSettings ...`? I've added it to build.sbt and it's giving an error `illegal start of simple expression`

**Comment by ikr0m** [comment]
Fixed the issue by adding following 2 lines to `build.sbt` file (I've got subprojects and I added those 2 lines only in main build.sbt)
```
scapegoatVersion in ThisBuild := "1.3.2"
scalaBinaryVersion in ThisBuild := "2.12"
```

Note: above fix requires sbt-scapegoat 1.0.5+ version
addSbtPlugin("com.sksamuel.scapegoat" %% "sbt-scapegoat" % "1.0.5")
